.flake8

```diff
@@ -3,3 +3,5 @@ max-line-length = 120
 exclude = .git/*, build/*, examples/*, venv/*, setup.py
 per-file-ignores =
     flask_rebar/__init__.py: F401 flask_rebar/*/__init__.py: F401
+select = B950
+extend-ignore = E501
```

Comment:
I don't have a problem with us switching to B950 instead of E501, but this change should be made in a separate PR.